service/src/test/java/bio/terra/landingzone/library/landingzones/AzureIntegrationUtils.java

```diff
@@ -12,11 +12,12 @@
 import org.assertj.core.util.Strings;
 
 public class AzureIntegrationUtils {
+
   /** Path to Azure properties file. */
   private static final String AZURE_PROPERTIES_PATH = "integration_azure_env.properties";
 
   /** Property prefix for properties in {@link #AZURE_PROPERTIES_PATH}. */
-  private static final String AZURE_PROPERTY_PREFIX = "integration.azure";
+  private static final String AZURE_PROPERTY_PREFIX = "workspace.azure.";
```

Comment:
Now this lines up with the properties `LandingZoneStairwayConfiguration` looks for.